### DISCUSSION TITLE
Create subfolders within a folder?

### DISCUSSION THREAD

**Discussion by jphilippe2017** [iptv-org/community#271] [Q&A] [answered]
Hello,

Is there a code please, to create subfolders within a folder for any links we have?

So as an example, if I were to install the news link from this website:
https://iptv-org.github.io/iptv/categories/news.m3u

A code that looks like this right now:

**#EXTINF:-1 tvg-id="OneAmericaNewsNetwork.us" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/G6e7Hcl.png" group-title="News",OAN (1080p)**
http://oneamericanews-roku-us.amagi.tv/playlist.m3u8

Should look like what if it has subfolders to it?

So lets say, if it had a main category called: US
subcategory called: english
and one more subcategory called: 1080p

But, a folder, that would have a subfolder within and one more subfolder within the last one (that was english).

Thank you!

**Comment by Dum4G** [comment, marked as answer]
Not possible since media players do not support that

**Comment by fershady** [comment]
You can always get the json format (https://iptv-org.github.io/iptv/channels.json) for any specific request.

**Comment by jphilippe2017** [comment]
**Fershady**, thank you for your reply, is there an m3u file for the link you sent plese: https://iptv-org.github.io/iptv/channels.json
Or, an Xtream api http(s) address, username and password to access this file.

Also, what would be the correct line of code for 1 channel (any) that is in a category folder, then in a subcategory folder and finally in one last subfolder, if created in json format? Can this be written in a .txt and saved as m3u file after that?
